example/jobflow/README.md

```diff
@@ -8,47 +8,16 @@ read design at [here](../../docs/design/jobflow).
 
 ### Prerequisites
 
-- docker: `18.06`
 - Kubernetes: >`1.17`
 
 ## startup steps
 
-build image from local
-```bash
-# get volcano and jobflow source code from github
-git clone http://github.com/volcano-sh/volcano.git
-git clone https://github.com/BoCloud/JobFlow.git
-
-# build image beyondcent/jobflow:v0.0.1 from local
-cd JobFlow
-make
-make docker-build
-```
-
-##### deploy JobFlow from [here](https://github.com/BoCloud/JobFlow#deploy)
-```bash
-kubectl apply -f https://raw.githubusercontent.com/BoCloud/JobFlow/main/deploy/jobflow.yaml
-```
-
-##### deploy Volcano from [here](https://volcano.sh/en/docs/installation/#install-with-yaml-files)
+### deploy Volcano from [here](https://volcano.sh/en/docs/installation/#install-with-yaml-files)
```

Comment:
Just keep all the installation ways （not only yaml） is ok.

Reply:
done